payments/card-payments/index.md

```diff
@@ -27,29 +27,29 @@ TODO: This page needs serious clean-up.
                       header="Site under development"
                       body="This section of the Developer Portal is under construction and should not be used to integrate against Swedbank Pay's APIs yet." %}
 
-{% include jumbotron.html body="**Credit Card Payments** is the most popular,
+{% include jumbotron.html body="**Card Payments** is the most popular,
 versatile and global way to initate a transaction with a customer. Choose
 between our **Seamless View**, **Redirect**, or **Direct** integration options." %}
 
 * When properly set up in your merchant/webshop site and the payer starts the
-purchase process, you need to make a `POST` request towards Swedbank Pay with
-your Purchase information. This will generate a payment object with a unique
-`paymentID`. You either receive a Redirect URL to a Swedbank Pay hosted
-page(Redirect integration) or a JavaScript source in
-response(Seamless View integration).
-* You need to [redirect][redirect] the payer's browser to that specified URL,
-or embed the script source on your site to create a Hosted View in an iFrame;
-so that she can enter the credit card details in a secure Swedbank Pay hosted
-environment.
+  purchase process, you need to make a `POST` request towards Swedbank Pay with
+  your Purchase information. This will generate a payment object with a unique
+  `paymentID`. You either receive a Redirect URL to a Swedbank Pay hosted
+  page(Redirect integration) or a JavaScript source in response(Seamless View
+  integration).
+* You need to [redirect][redirect] the payer's browser to that specified URL, or
+  embed the script source on your site to create a Hosted View in an `iframe`;
+  so that she can enter the credit card details in a secure Swedbank Pay hosted
+  environment.
 * Swedbank Pay will handle 3-D Secure authentication when this is required.
-* Swedbank Pay will redirect the payer's browser to - or display directly in
-the iFrame - one of two specified URLs, depending on whether the payment session
- is followed through completely or cancelled beforehand. Please note that both a
-  successful and rejected payment reach completion, in contrast to a cancelled
+* Swedbank Pay will redirect the payer's browser to - or display directly in the
+  `iframe` - one of two specified URLs, depending on whether the payment session
+  is followed through completely or cancelled beforehand. Please note that both
+  a successful and rejected payment reach completion, in contrast to a cancelled
   payment.
 * When you detect that the payer reach your `completeUrl` , you need to do a
-`GET` request, containing the `paymentID` generated in the first step, to
-receive the state of the transaction.
+  `GET` request, containing the `paymentID` generated in the first step, to
+  receive the state of the transaction.
 
 ## Screenshots
```

Comment:
Please move the screenshots to a sub-section so it isn't the most prominent header in the file.